File: T4. C# BASIC SYNTAX - EXERCISES (Java)/src/P04_BeverageLabels.java
import java.text.DecimalFormat;
import java.util.Scanner;

public class P04_BeverageLabels {
    public static void main(String[] args) {

        Scanner in = new Scanner(System.in);
        DecimalFormat df = new DecimalFormat("#.##");

        String name = in.nextLine();
        double volume = Double.parseDouble(in.nextLine());
        double energy = Double.parseDouble(in.nextLine());
        double sugar = Double.parseDouble(in.nextLine());

        double energyVol = (volume * energy) / 100;
        double sugarVol = (volume * sugar) / 100;

        System.out.printf("%.0fml %s:\n", volume, name);
        System.out.println(df.format(energyVol) + "kcal, " + df.format(sugarVol) + "g sugars");
    }
}

File: T4. C# BASIC SYNTAX - EXERCISES/P04. BeverageLabels.cs
using System;

namespace P04._BeverageLabels
{
    class Program
    {
        static void Main(string[] args)
        {
            string name = Console.ReadLine();
            double volume = double.Parse(Console.ReadLine());
            double energy = double.Parse(Console.ReadLine());
            double sugar = double.Parse(Console.ReadLine());

            double energyVol = (volume * energy) / 100;
            double sugarVol = (volume * sugar) / 100;

            Console.WriteLine($"{volume}ml {name}:");
            Console.WriteLine($"{energyVol}kcal, {sugarVol}g sugars");
        }
    }
}
